virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -538,3 +538,38 @@ def create_remote_disk_by_same_metadata(vm, params):
 
     remote_session.close()
     return blk_source
+
+
+def update_vm_disk_target(vm_name, target_dict, disk_index=0):
```

Comment:
@dzhengfy 
There is existed method libvirt.set_vm_disk(vm, params, tmp_dir) to achieve update disk.
So is there other requirements that set_vm_disk can not satisfy ,and therefore create this new one?

Reply:
@chunfuwen The main reasons are below:
1. set_vm_disk() is too complex and readability is not good any more. 
2. I need to support new attribute in <target rotation_rate='' >, but set_vm_disk() does not support. I would not like to add new support to that function.
So I need a new style function to update disk target attributes, instead of hard coding some attributes in the function

Reply:
@dzhengfy, here is what we need do if update existed disk.
And you need update target, it also allow you update disk_target and disk_target_bus accordingly.

    disk_target = params.get("disk_target", 'vda')
    disk_target_bus = params.get("disk_target_bus", "virtio")
```
            params.update({'disk_source_name': replace_disk_image,
                           'disk_type': 'file',
                           'disk_format': 'qcow2',
                           'disk_source_protocol': 'file'})
            libvirt.set_vm_disk(vm, params, tmp_dir)
```

Reply:
@chunfuwen  Update to use set_vm_disk(). Please review again. Thanks.